flask_appbuilder/security/sqla/manager.py

```diff
@@ -214,7 +214,7 @@ def add_user(
         except Exception as e:
             log.error(c.LOGMSG_ERR_SEC_ADD_USER.format(str(e)))
             self.get_session.rollback()
-            return False
+            return None
```

Comment:
Agree! would be nice to add type annotations to add_user since we are here

Reply:
added as "->User" if you are OK with it.

Reply:
removed "->User", as the  user model can be changed at run time